new/LIT7099QeneEffoHalloka.xml

```diff
@@ -0,0 +1,83 @@
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. It has four lines, meaning
+                    that it can be classified as a <foreign xml:lang="gez">kəbr yəʾəti qəne</foreign> in either the mode
+                    <foreign xml:lang="gez">gəʿz</foreign> or the mode <foreign xml:lang="gez">ʿəzl</foreign> according to
+                    <bibl><ptr target="bm:Guidi1900Qene"/></bibl>. 
+                    It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD">Gabra Ləʿul</persName> 
+                    in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="KebrYeeti" cert="medium"/>
+                </keywords>
+            </textClass>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </t:langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-10-14">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7099QeneEffoHalloka" passive="UppQ1524o"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7099QeneEffoHalloka" passive="PRS14574GabraLeul"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fol. 42r.
+                    Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .</note>
+                <ab>
+                    <l n="1"><choice><sic>አ</sic><corr resp="JK">እ</corr></choice>ፎ፡ ሀሎከ፡ በጊዜ፡ 
+                        ትቤ<del rend="overUnderlined">ሳ</del><add place="above">ላ</add>፡ ኦሪተ፡ ሙሴ፡ ቅዱስ፤</l>
```

Comment:
This letter was over- and underlined and crossed out. It has been deleted twice. A del can have a rend, but you can give two del:  
`<del rend="strikethrough"><del rend="overUnderlined">ሳ</del></del><add place="above">ላ</add>`
However, I do not know if the result will be useful and nice to look at. So it might be better to leave it as it is.

Reply:
I added two rend now, and the schema didn't complain :)